client/composants/avis/AddRating.js

```diff
@@ -45,7 +45,7 @@ export default function AddRatingComposant({ prevStep }) {
         remarkNegative: remarkNegative || null,
         qualityOfLife: data?.qualityOfLife || null,
       });
-      setResponse(responseToFront.save.message);
+      setResponse(responseToFront);
```

Comment:
Only "response" here, api can be called by another thing :)
```suggestion
      setResponse(response);
```